### PR TITLE
Fixed issue #18929: Cannot edit data notification entries

### PR DESCRIPTION
**Pull request by gabrieljenik**
- Fix path for JS
- Fix path for logo

### CODE REVIEW

**Comment by Shnoulle** [comment]
We don't use assetmanager for other adminscripts ?

**Comment by gabrieljenik** [comment]
> We don't use assetmanager for other adminscripts ?

Not sure what you mean sorry

**Comment by Shnoulle** [comment]
> > We don't use assetmanager for other adminscripts ?
> 
> Not sure what you mean sorry

Something like that : `App()->getClientScript()->registerScriptFile(App()->assetManager->publish(App()->getConfig('adminscripts')) . 'failedEmail.js');`

Then when you update LimeSurvey : you are sure this file was updated

**Comment by Shnoulle** [comment]
> > > > We don't use assetmanager for other adminscripts ?
> > > 
> > > 
> > > Not sure what you mean sorry
> > 
> > 
> > Something like that : `App()->getClientScript()->registerScriptFile(App()->assetManager->publish(App()->getConfig('adminscripts')) . 'failedEmail.js');`
> > Then when you update LimeSurvey : you are sure this file was updated in browser cache.
> 
> `registerScriptFile` already handles the asset publishing automatically. image

Oh ! I din't know this …

Then sometimes : we do assets 2 times …

File: application/views/layouts/adminmenu.php [hunk 1]
```diff
@@ -15,7 +15,7 @@
             <span class="navbar-toggler-icon"></span>
         </button>
         <a class="navbar-brand" href="<?php echo $this->createUrl("/admin/"); ?>">
-            <img src="/assets/images/logo-icon-white.png" height="34" class="d-inline-block align-bottom" alt="">
+            <img src="<?= Yii::app()->baseUrl ?>/assets/images/logo-icon-white.png" height="34" class="d-inline-block align-bottom" alt="">
```
Comment (Shnoulle) [inline application/views/layouts/adminmenu.php:18]: Use assetManager here ?

**Review by Shnoulle — APPROVED** [review]

**Comment by Shnoulle** [comment]
No issue in code, maybe find a way to use assetmanager for logo too.

File: application/controllers/FailedEmailController.php [hunk 2]
```diff
@@ -48,7 +48,7 @@ public function actionIndex(): void
             App()->user->setState('pageSize', (int) $pageSize);
         }
 
-        App()->getClientScript()->registerScriptFile('/application/views/failedEmail/javascript/failedEmail.js', LSYii_ClientScript::POS_BEGIN);
+        App()->getClientScript()->registerScriptFile(App()->getConfig('adminscripts') . 'failedEmail.js', LSYii_ClientScript::POS_BEGIN);
```
Comment (olleharstedt) [inline application/controllers/FailedEmailController.php:51]: Looks like a missing folder? `failedEmail` folder.

Comment (olleharstedt) [reply application/controllers/FailedEmailController.php:51]: Oh no, sorry, the file was moved.

**Pull request merged**